app/Models/User.php

```diff
@@ -79,4 +79,13 @@ public function reviews()
     {
         return $this->hasMany(Review::class, 'user_id');
     }
+    public function getCoursesAttribute()
```

Comment:
attribute này dùng ở đâu nhỉ?

Reply:
à cái hàm này để lấy ra các course mà 1 user đã join, hôm trước em viết nhầm nhưng mà hình như sau này đến trang profile hình như cần dùng để render ra view nên em không xóa đi ạ.

Reply:
ở đó gọi `$user->courses()` được không nhỉ?

Reply:
à vâng em suy nghĩ lại thì chỗ đó chắc là dùng scope để lọc ra các course mà user đã join và dùng foreach in ra tất cả logo và title thôi ạ ^^